rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js

```diff
@@ -706,8 +705,9 @@ function createDirectiveTable(isTopLevel, isNodeView, contextPath) {
           editLink.click(function(e) {e.stopPropagation();});
           editLink.append(editIcon);
           editLink.addClass("reportIcon");
-
           $(nTd).append(editLink);
+          var policyMode = oData.policyMode ? oData.policyMode : "";
```

Comment:
Maybe you should keep undefined / treat undefined (find it better to deal with than empty String